onnx/validate.py

```diff
@@ -0,0 +1,48 @@
+"""onnx checker
+
+This implements graphalities that allows us to check whether a serialized
```

Comment:
graphalities => modalities? I suspect some old renaming of model to graph may be the cause!